CHANGELOG.md

```diff
@@ -5,21 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0] - 02-02-23
 
 ### Added
-* Added timeout condition for the validation of master presence in 
+* Add timeout condition for the validation of master presence in 
   replicaset and for the master connection (#95).
 * Support Cartridge clusterwide configuration for `crud.cfg` (#332).
 
 ### Changed
 * **Breaking**: forbid using space id in `crud.len` (#255).
 
 ### Fixed
-* Added validation of the master presence in replicaset and the 
+* Add validation of the master presence in replicaset and the 
   master connection to the `utils.get_space` method before 
   receiving the space from the connection (#331).
-* Fixed fiber cancel on schema reload timeout in `call_reload_schema`.
+* Fix fiber cancel on schema reload timeout in `call_reload_schema` (PR #337).
```

Comment:
As far as I remember, the [commit](https://github.com/tarantool/crud/pull/336/commits/906a2d3d25c734562c1899366b18e86e4ff6a62c) belongs to #336

Reply:
Thank you, fixed